test/integration/local/test_python_service.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import os
+import subprocess
+import sys
+import time
+
+import pytest
+
+import requests
+
+PING_URL = 'http://localhost:8080/ping'
+INVOCATIONS_URL = 'http://localhost:8080/invocations'
+
+
+@pytest.fixture(scope='session', autouse=True)
+def volume(tmpdir_factory):
+    try:
+        with open('test/integration/local/inference_test_example.py') as example:
+            with open('test/resources/models/inference.py', 'w') as custom_code:
+                for line in example:
+                    custom_code.write(line)
+
+        model_dir = os.path.abspath('test/resources/models')
+        subprocess.check_call(
+            'docker volume create --name model_inference_volume --opt type=none '
+            '--opt device={} --opt o=bind'.format(model_dir).split())
+        yield model_dir
+    finally:
+        os.remove('test/resources/models/inference.py')
```

Comment:
writing to source

Reply:
I used tmpdir_factory instead of tmpdir because tmpdir cannot be used in session scope. And with tmpdir_factory, it requires a non-existing directory to be the basetmpdir, so I put inference.py to test/resources/models/**script**/inference.py